lib/takeaway.rb

```diff
@@ -0,0 +1,31 @@
+# require_relative 'menu'
+
+class Takeaway
+  attr_reader :order, :total
+
+  def initialize(meals:)
+    @order = []
+    @total = 0
+    @menu = meals
+  end
```

Comment:
Nice use of dependency injection to make your Takeaway class be able to use your Menu class!